lib/Bio/KBase/HandleMngr/HandleMngrServer.pm

```diff
@@ -0,0 +1,751 @@
+package Bio::KBase::HandleMngr::HandleMngrServer;
+
```

Comment:
Do we want to commit the Server.pm file? I think it may be generated by kb-sdk compile.

Reply:
See comment for the .psgi file